test/testclass.cpp

```diff
@@ -5552,14 +5552,16 @@ class TestClass : public TestFixture {
                    "int MixerParticipant::GetAudioFrame() {\n"
                    "    return 0;\n"
                    "}");
-        ASSERT_EQUALS("[test.cpp:4] -> [test.cpp:2]: (performance, inconclusive) Technically the member function 'MixerParticipant::GetAudioFrame' can be static (but you may consider moving to unnamed namespace).\n", errout.str());
+        // this code is invalid so a false negative is OK
+        TODO_ASSERT_EQUALS("[test.cpp:4] -> [test.cpp:2]: (performance, inconclusive) Technically the member function 'MixerParticipant::GetAudioFrame' can be static (but you may consider moving to unnamed namespace).\n", "", errout.str());
```

Comment:
Imho it's better that we remove the assertion. In theory it might be better to write a warning but it feels like in practice it's better to not try to handle invalid code. The comment you added is valueable.